core/accounting/src/chart_of_accounts/ledger/mod.rs

```diff
@@ -71,9 +68,9 @@ impl ChartLedger {
             .clone()
             .metadata
             .unwrap_or_else(|| serde_json::json!({}));
-        AccountingClosingMetadata::update_metadata(
+        crate::accounting_period::ClosingMetadata::update_metadata(
```

Comment:
Small nit, can potentially include this in the `use crate..` statement in imports so we can quickly see all which crates are being used at the top of the file 
```suggestion
        ClosingMetadata::update_metadata(
```